gradle/project.gradle.kts

```diff
@@ -170,6 +175,8 @@ tasks {
         description = "Links $dirJdk to JDK"
         onlyIf { !dirJdk.exists() }
         doFirst {
+            // if the symbolic link is invalid, "exist" returns false, but it still blocks the creation of the new new link, so it has to first be deleted
```

Comment:
Comments above the code make it harder to reason about code structure. Can we use comments right of the relevant code?

Also, can be radically shortened. Let the comment act as a hint, not a full blown explanations. For example:
```
dirJdk.delete()    // delete invalid symbolic link
```